gh-pages/web-features.md

```diff
@@ -0,0 +1,37 @@
+---
+permalink: web-features
+---
+
+# Web features
+
+web-features is an effort to build a shared catalog of features of the web platform. By creating a common list of features and their definitions, web-features aims to improve understanding of what web developers get and want from the web. web-features does this by:
+
+- Creating feature definitions, which identify and describe capabilities of the web.
+- Generating Baseline support data, which summarizes the availability of web features across key browsers and releases.
+- Publishing the [web-features npm package](https://www.npmjs.com/package/web-features), which bundles feature identifiers with Baseline statuses.
+
+## What is a feature?
+
+Think of features as individual components of the web platform that web developers can use to achieve specific tasks.
+
+As such, a feature has no specific size. A feature might cover a single CSS property, an entire JavaScript APIs with multiple interfaces, methods, and properties, or a combination of CSS properties, HTML elements, and JavaScript APIs. As long as a web developer would use the feature to achieve a specific goal, it's a feature.
+
+For example, the `fetch()` API is a feature, the `:has()` CSS pseudo-class function too, and so is the Web Audio API.
+
+## How is a feature described?
```

Comment:
I would actually remove this section. As a casual reader, I'm not interested in these internal details. I'm also not interested in this as a consumer of the package. If I'm looking to contribute, then yes, but in that case I would much rather go and read the contributor docs on the repo directly.

Reply:
removed it